src/cmd/prepare.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/defenseunicorns/zarf/src/config/lang"
 	"github.com/defenseunicorns/zarf/src/pkg/message"
 	"github.com/defenseunicorns/zarf/src/pkg/packager"
+	validator "github.com/defenseunicorns/zarf/src/pkg/packager/validator"
```

Comment:
```suggestion
	"github.com/defenseunicorns/zarf/src/pkg/packager/validator"
```
You don't need to overwrite the name when it is the same name